src/api/v1beta1/feature_flags.go

```diff
@@ -160,9 +160,9 @@ func (dk *DynaKube) FeatureDisableReadOnlyOneAgent() bool {
 // FeatureEnableActivegateRawImage is a feature flag to specify if the operator should
 // fetch from cluster and set in ActiveGet container: tenant UUID, token and communication endpoints
 // instead of using embedded ones in the image
-// Defaults to false
+// Defaults to true
```

Comment:
wouldn't it be nicer to have a `disable` flag which if set to `true` we disable it ?
I think the other flags follow this logic

Reply:
This was my initial idea but then I realized we don't want to support two annotations for the same thing:
- `feature-enable-activegate-raw-image` (old one, released in `release-0.5`)
- `feature-disable-activegate-raw-image` (new)

Reply:
it wasn't released in `v0.5` so we should go with the disable flag to be consistent